core/trashcan.js

```diff
@@ -609,7 +609,8 @@ class Trashcan extends DeleteArea {
    * @private
    */
   onDelete_(event) {
-    if (this.workspace_.options.maxTrashcanContents <= 0 || event.type !== eventUtils.BLOCK_DELETE) {
+    if (this.workspace_.options.maxTrashcanContents <= 0 ||
+        event.type !== eventUtils.BLOCK_DELETE) {
```

Comment:
I'd even consider using `instanceof` here too just for consistency, though it's less beneficial when we don't need the type-narrowing.